library/src/main/java/com/mapzen/android/MapzenMap.java

```diff
@@ -394,4 +395,129 @@ public boolean isSimultaneousGestureAllowed(TouchInput.Gestures first,
             TouchInput.Gestures second) {
         return mapController.isSimultaneousGestureAllowed(first, second);
     }
+
+    /**
+     * Draws two pins on the map. The start pin is active and the end pin is inactive.
+     * @param start
+     * @param end
+     */
+    public void drawRoutePins(LngLat start, LngLat end) {
+        overlayManager.drawRoutePins(start, end);
+    }
+
+    /**
+     * Clears the start and end pins from the map.
+     */
+    public void clearRoutePins() {
+        overlayManager.clearRoutePins();
+    }
+
+    /**
+     * Draws a dropped pin on the map at the point supplied.
+     * @param point
+     */
+    public void drawDroppedPin(LngLat point) {
+        overlayManager.drawDroppedPin(point);
+    }
+
+    /**
+     * Clears the dropped pin from the map.
+     */
+    public void clearDroppedPin() {
+        overlayManager.clearDroppedPin();
+    }
+
+    /**
+     * Draws a search result on the map at the point supplied. The pin will be active.
+     * @param point
+     */
+    public void drawSearchResult(LngLat point) {
+        drawSearchResult(point, true);
+    }
+
+    /**
+     * Draws a search result on the map at the point supplied.
+     * @param point
+     * @param active
+     */
+    public void drawSearchResult(LngLat point, boolean active) {
+        overlayManager.drawSearchResult(point, active);
+    }
+
+    /**
+     * Draws search results on the map. All pins are displayed as active.
+     * @param points
+     */
+    public void drawSearchResults(List<LngLat> points) {
+        int index = 0;
+        for (LngLat point : points) {
+            overlayManager.drawSearchResult(point, true, index);
+            index++;
+        }
+    }
+
+    /**
+     * Draws search results on the map. All pins will be inactive except for the one at the active
+     * index supplied.
+     * @param points
+     * @param activeIndexes
+     */
+    public void drawSearchResults(List<LngLat> points, int... activeIndexes) {
+        int index = 0;
+        for (LngLat point : points) {
+            boolean status = false;
+            for (int i = 0; i < activeIndexes.length; i++) {
+                if (activeIndexes[i] == index) {
+                    status = true;
+                    break;
+                }
+            }
+            overlayManager.drawSearchResult(point, status, index);
+            index++;
+        }
+    }
+
+    /**
+     * Clears search result from the map.
+     */
+    public void clearSearchResult() {
```

Comment:
I understand why we have both `clearSearchResult()` and 'clearSearchResults()` for completeness but given they have the same implementation perhaps we can delete one to avoid confusion.